test/dom/element.test.js

```diff
@@ -208,6 +208,79 @@ describe('Document', () => {
 		expect(doc.childNodes.toString()).toBe(`<A/><B/><C/>`)
 	})
 
+	describe('element traversal', () => {
+		it('initialization', () => {
+			const doc = new DOMParser().parseFromString(`<A/>`)
+			const aElement = doc.firstChild
+
+			expect(aElement.childElementCount).toBe(0)
+			expect(aElement.firstElementChild).toBeNull()
+			expect(aElement.lastElementChild).toBeNull()
+			expect(aElement.previousElementSibling).toBeNull()
+			expect(aElement.nextElementSibling).toBeNull()
+		})
+
+		it('append one element and remove', () => {
+			const doc = new DOMParser().parseFromString(`<A></A>`)
+			const aElement = doc.firstChild
+			const bElement = doc.createElement('B')
+			aElement.appendChild(bElement)
+
+			expect(aElement.childElementCount).toBe(1)
+			expect(aElement.firstElementChild === bElement).toBe(true)
+			expect(aElement.lastElementChild === bElement).toBe(true)
```

Comment:
Minor: This could be written as
```suggestion
			expect(aElement.firstElementChild).toBe(bElement)
			expect(aElement.lastElementChild).toBe(aElement.firstElementChild)
```
What do you think about this approach?